app/models/video.rb

```diff
@@ -1,4 +1,5 @@
 class Video < ApplicationRecord
+  validates_presence_of :title, :description, :thumbnail
```

Comment:
Feels like Mod 2 all over again! It'll probably work out nicely to have these validated. It'll definitely make sad path a little easier to deal with.

Reply:
had to take these out or all tests fail :P